### PR TITLE
fix: improve cache settings

### PR DESCRIPTION
**Pull request by kayman-mk**
## Description

A caching period of one day seems to be too short as default. This has been adjusted to 7 days to allow some caching.

## Migrations required

No

## Verification

No verification done. No relevant parts changed.

### CODE REVIEW

**Comment by kayman-mk** [comment]
Shall the caching period reduced to 5 days may be? Usually you work with versioned artifacts so you could cache 100 days but it doesn't make sense to have all old (and hopefully unused) versions in the cache.

File: modules/cache/variables.tf [hunk 1]
```diff
@@ -54,7 +54,7 @@ variable "cache_lifecycle_clear" {
 variable "cache_lifecycle_prefix" {
   description = "Object key prefix identifying one or more objects to which the clean up rule applies."
   type        = string
-  default     = "runner/"
+  default     = "project/"
```
Comment (npalm) [inline modules/cache/variables.tf:57]: The root directory is runner in the bucket, for that reason I had set the default to runner. What is the issue here?

Comment (npalm) [reply modules/cache/variables.tf:57]: Checking my current deployment, to see if it is not cleared. Will update in a few days.

Comment (kayman-mk) [reply modules/cache/variables.tf:57]: Strange. The root directory is `project/` in my case and I am using the default values only.

The lifecycle rule says `runner/` thus no clean up is done here.

I will also doublecheck my code.

Comment (npalm) [reply modules/cache/variables.tf:57]: Just doubled checked, deployed the default example (examples dir) and cache files are stored in runners/<id>/project. So the root dir in my case is runner

Comment (kayman-mk) [reply modules/cache/variables.tf:57]: Strange. I will revert this change.

**Comment by kayman-mk** [comment]
@npalm What do you think about increasing the default caching period? PR to be closed or to be merged?

**Comment by npalm** [comment]
I personally not a fan of long caching, since it can introduce a false sens of things keep working due to caching. For that reason I have set the caching short. Of course it the writer of the pipeline to think about the impact of caching. So setting the default higher is fine by me as well. In the end it is configurable

**Comment by kayman-mk** [comment]
As it is configurable, I will close this one.

**Pull request closed without merging**